src/test/java/GitHubTest.java

```diff
@@ -1,2 +1,40 @@
-package PACKAGE_NAME;public class GitHubTest {
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class GitHubTest extends BaseTest {
+
+    @Test
+    public void validateLogoOnTheLoginPageIsDisplayed() {
+        HomePage homePage = new HomePage(driver);
+        Assertions.assertTrue(homePage.goToLoginPage().getLogo().isDisplayed());
+    }
+
+    @Test
+    public void validateAnUserCanLoginSuccessfully() {
+        HomePage homePage = new HomePage(driver);
+        Assertions.assertEquals("Home", homePage.goToLoginPage().loginSuccessful("work46341@gmail.com",
+                "work46341@gmail.com").getHeaderLocator().getText());
+    }
+
+    @Test
+    public void validateFailedLogin() {
+        HomePage homePage = new HomePage(driver);
+        homePage.goToLoginPage().loginFailed("work4634@gmail.com", "work46341@gmail.com").
+                validateErrorMessage("Incorrect username or password.");
+    }
+
+    @Test
+    public void validateLogoOnTheIssuesPageIsDisplayed() {
+        HomePage homePage = new HomePage(driver);
+        Assertions.assertTrue(homePage.goToLoginPage().loginSuccessful("work46341@gmail.com",
+                "work46341@gmail.com").goToIssuesPage().getIssuesLogLocator().isDisplayed());
+    }
+
+    @Test
```

Comment:
To test functionality on the Issues page we should check if there is possibility to create a new issue and check if it has been created successfully

Reply:
e.g., add test validateThatAnUserCanCreateANewIssue